xrpl/models/transactions/vault_create.py

```diff
@@ -0,0 +1,109 @@
+"""Represents a VaultCreate transaction on the XRP Ledger."""
+
+from dataclasses import dataclass, field
+from enum import Enum
+from typing import Dict, Optional, Union
+
+from typing_extensions import Self
+
+from xrpl.models.currencies import Currency
+from xrpl.models.flags import FlagInterface
+from xrpl.models.required import REQUIRED
+from xrpl.models.transactions.transaction import Transaction
+from xrpl.models.transactions.types import TransactionType
+from xrpl.models.utils import KW_ONLY_DATACLASS, require_kwargs_on_init
+
+_MAX_DATA_LENGTH = 256 * 2
+_MAX_MPTOKEN_METADATA_LENGTH = 1024 * 2
+_MAX_DOMAIN_ID_LENGTH = 32 * 2
```

Comment:
```suggestion
VAULT_MAX_DATA_LENGTH = 256 * 2
VAULT_MAX_DOMAIN_ID_LENGTH = 32 * 2
_VAULT_MAX_MPTOKEN_METADATA_LENGTH = 1024 * 2
```
'_' should be removed from the ones that are also used in `VaultSet`